lib/xiaomi.js

```diff
@@ -502,6 +502,11 @@ const numericAttributes2Payload = async (msg, meta, model, options, dataObject)
                 payload.buzzer_manual_alarm = value === 1;
             }
             break;
+        case '320':
+            if (['MCCGQ13LM'].includes(model.model)) {
+                payload.cover = {0: 'Close', 1: 'Open'}[value];
```

Comment:
Please rename this to `battery_cover` and make the values `CLOSE`/`OPEN`

Reply:
Done.  Renamed to battery_cover and made the values CLOSE/OPEN